yt/geometry/particle_deposit.pxd

```diff
@@ -14,12 +14,13 @@ import numpy as np
 cimport cython
 from libc.math cimport sqrt
 from libc.stdlib cimport free, malloc
-from numpy.math cimport PI as NPY_PI
 
 from yt.utilities.lib.fp_utils cimport *
 
 from .oct_container cimport Oct, OctreeContainer
 
+cdef extern from "numpy/npy_math.h":
+    double NPY_PI
```

Comment:
To see why this change was needed, see https://cython.readthedocs.io/en/latest/src/changes.html#other-changes

> The previously shipped NumPy C-API declarations (cimport numpy) were removed. NumPy has been providing version specific declarations for several versions now. (Github issue [#5842](https://github.com/cython/cython/issues/5842))

Reply:
(`NPY_2_PI` is also available)